### DISCUSSION TITLE
question regarding pluing loading

### DISCUSSION THREAD

**Discussion by bugsbugsbux** [willothy/flatten.nvim#27] [Q&A] [answered]
In the README you show an install method "to avoid loading plugins in the guest". I thought this plugin prevents a new guest and opens the file in the host? Does it reload my plugins?

Moreover, this way of setting the plugin up with lazy.nvim is not very convenient because then flatten.nvim appears as plugin to clean in the lazy ui and has to be reinstalled every time I clean some other plugins. How about putting the check in lazy.nvim's global condition (defaults.cond) instead? Or is there another way?

Awesome plugin, kind regards

**Comment by willothy** [comment, marked as answer]
The guest opens for a short amount of time to send an RPC request to the host, not loading other plugins just makes that happen faster. It's definitely not convenient, I agree. I'll look into a better method. However, if you don't want to use that, it's not strictly necessary. It just might add a bit more latency to opening files depending on the load time.

Thanks for using the plugin!

**Comment by bugsbugsbux** [comment]
I put this into my lazy.nvim settings.

```lua
{
    defaults = {
        cond = function(plugin)
            if os.getenv('NVIM') ~= nil then
                if plugin[1] ~= 'willothy/flatten.nvim' then
                    return false
                end
                return true
            end
            return true
        end,
    }
}
```
